### ISSUE TITLE
Docker container restart issue

### ISSUE THREAD

**Issue by modem7** [crowdsecurity/cs-cloudflare-worker-bouncer#50]
When I try to restart the container, it tries to clean up the worker but fails. 

```
time="2024-11-08T19:53:30Z" level=info msg="Using API key auth"
time="2024-11-08T19:53:31Z" level=info msg="Cleaning up existing workers" account=myaccount@mydomain.com
time="2024-11-08T19:53:33Z" level=fatal msg="unable to cleanup existing workers: remove namespace: 'namespace has associated scripts: crowdsec-cloudflare-worker-bouncer' (10052) for account myaccount@mydomain.com"
```

```yaml
  # CrowdSec Bouncer - Cloudflare
  # sudo docker exec crowdsec cscli bouncer add cloudflare-worker-bouncer
  # To recover: docker run --rm -it -v "$(pwd)"/cfg.yaml:/etc/crowdsec/bouncers/crowdsec-cloudflare-worker-bouncer.yaml --name WorkerBouncerRecovery 'crowdsecurity/cloudflare-worker-bouncer' -d
  cloudflare-worker-bouncer:
    image: crowdsecurity/cloudflare-worker-bouncer
    container_name: Crowdsec-Cloudflare-Worker-Bouncer
    networks:
      pihole:
        ipv4_address: '172.22.0.125'
    # ports:
    #  - "2112:2112"
    volumes:
      - $USERDIR/Crowdsec/cloudflare-worker-bouncer/cfg.yaml:/etc/crowdsec/bouncers/crowdsec-cloudflare-worker-bouncer.yaml
      # - $USERDIR/Crowdsec/cloudflare-worker-bouncer/cf-bouncer:/cf-bouncer
    # labels:
    #   - "backup"
    security_opt:
      - no-new-privileges:true
    logging:
      driver: "local"
      options:
        max-size: 10m
        max-file: "3"
    restart: always
    mem_limit: 500m
    mem_reservation: 50m
    depends_on:
      - crowdsec
```

![image](https://github.com/user-attachments/assets/905e50d9-6582-4fc0-892f-56bf833ac00b)

**Comment by blotus** [comment]
Hello,

This seems to come from a change on cloudflare side.

I've just pushed a fix that resolves the issue.

I've made a new pre-release (v0.0.12) with the fix, the docker image will be available in about 30 minutes (with the `v0.0.12` tag, `latest` will be updated when we turn it into an actual release, which will not happen before next week).

If you switch to the v0.0.12 image for testing, note that this new version also includes support for [usage metrics](https://docs.crowdsec.net/docs/next/observability/usage_metrics/) and now creates a D1 database to store them.
Because your token does not have any permissions on D1, you'll see a warning when the bouncer starts, but you can safely ignore it (you can refer to this documentation PR if you want to set it up: https://github.com/crowdsecurity/crowdsec-docs/pull/651/files)

**Comment by modem7** [comment]
Hi @blotus - Thank you very much for the fast response and turnaround! 

All confirmed working on `crowdsecurity/cloudflare-worker-bouncer:v0.0.12`.

Sincerely appreciated!
